parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
I think functions should only do one of these two things:

1. Have a side-effect
2. Return a value

This isn't always true.  Writing to the logs or updating a random number generator's state are side-effects, and are sometimes intermixed in functions that return values.  Still, I think we should avoid doing both when possible and in this case, it seems very possible.

This is a pretty important concept, so I want to hear what @andrewemark and @yujanshrestha think too!